### PR TITLE
conversion to syled-components

### PR DESCRIPTION
**Pull request by Davit-Avakian**

### CODE REVIEW

**Review by NareOP — CHANGES_REQUESTED** [review]
Overall feedback:
the code and file structures look good to me, I can see detailed designing approaches to the components. Excellent work on that! 

Please also consider:

- the naming of the PR and commits and branch names, don't overload one commit with lots of changes do it separately, so it'll be easy to organize, it'll be understandable for other developers and modifications can be seen independently.
- add eslint and prettier to the project, so you can keep your code formatted, and avoid minor mistakes.
- keep one type of metric in your code, use just rem in your styling (for small sizes like 3px, you can keep the px, it's much understandable)
- we usually keep images and fonts in src/assets folder, please keep that
- for the class namings of CSS classes we use kebab-case
- instead of relative paths use path aliases, fyi read [this](https://dev.to/rfornal/create-path-aliases-in-react-2oid) article

**Review by yaser-op — CHANGES_REQUESTED** [review]
too many files, what are these for?
image
image

File: src/App.js [hunk 1]
```diff
@@ -1,56 +1,78 @@
...
+            </MainWrapper>
 
-          <div className="Second-Wrapper">
-            <div className="Main-Content">
-              <Trending />
-              <Coverage />
-              <TeamsInDays />
-              <AroundNba />
-            </div>
+            <SecondWrapper>
```
Comment (yaser-op) [inline src/App.js]: rename to more appropriate name; something like `SecondaryWrapper`

File: src/components/AroundNba/index.js [hunk 2]
```diff
@@ -24,34 +27,37 @@ const images = {
...
-              <div>
-                <span className="title">
-                  {title}
-                </span>
-                <span className="description">{description}</span>
-                <span className="date">{date}</span>
-              </div>
+            <li key={id}>
+              <a href="">
+                <img src={images[image]} />
```
Comment (yaser-op) [inline src/components/AroundNba/index.js]: in a real app this is not the way to do it, the item that you are showing will contain the image url, update the `aroundNba` object to add the image url there and in the view render the image received .

applies everywhere else

File: src/components/Coverage/styles.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components';
+
+export const CoverageItem = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: row !important;
```
Comment (yaser-op) [inline src/components/Coverage/styles.js]: avoid using `!important`

File: src/components/Coverage/styles.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  cursor: pointer;
+
+  img {
+    height: 130px;
+  }
+`;
+
+export const CoverageContainer = styled.div`
+  height: fit-content;
+  background-color: #ffff;
```
Comment (yaser-op) [inline src/components/Coverage/styles.js]: this is used in several styles, move the colors to a global file and use it everywhere

**Review by yaser-op — CHANGES_REQUESTED** [review]
include video recording or screenshots for all pages desktop & mobile

File: src/components/Footer/index.js [hunk 5]
```diff
@@ -22,79 +22,71 @@ const Footer = () => {
...
-            </li>
+            {['League Pass', 'Customer Support'].map((el) => {
+              return (
+                <li key={el}>
+                  <a>{el}</a>
+                </li>
+              );
+            })}
           </ul>
         </div>
```
Comment (NareOP) [inline src/components/Footer/index.js]: I can see the same structure a few times, so make this dynamic as well

File: src/components/AroundNba/index.js [hunk 6]
```diff
@@ -1,57 +1,37 @@
...
-
-              <div>
-                <span className="title">
-                  {title}
-                </span>
-                <span className="description">{description}</span>
-                <span className="date">{date}</span>
-              </div>
+            <li key={id}>
+              <a href="">
```
Comment (NareOP) [inline src/components/AroundNba/index.js]: what's the purpose of a tag here?

File: src/components/AroundNba/index.js [hunk 7]
```diff
@@ -1,57 +1,37 @@
...
+                <img src={image} />
+              </a>
+              <NewsItem>
+                <span>{title}</span>
+                <span>{description}</span>
+                <span>{date}</span>
+              </NewsItem>
```
Comment (NareOP) [inline src/components/AroundNba/index.js]: please take out this element, I saw something like this in another file too, make it shared element

File: src/components/Header/index.js [hunk 8]
```diff
@@ -1,241 +1,209 @@
...
-                  <li>DraftKings DFS</li>
-                  <li>Starting Lineups</li>
+                  {NbaLinks.map((el) => {
+                    return <li key={el}>{el}</li>;
+                  })}
                 </ul>
-              </div>
-            </div>
+              </PopUp>
+            </Link>
```
Comment (NareOP) [inline src/components/Header/index.js]: same structure, do it with map and do the same for Menu component

File: src/components/SocialMedias/index.js [hunk 9]
```diff
@@ -38,7 +43,7 @@ const SocialMedias = () => {
           <img src={twitch} />
         </a>
```
Comment (NareOP) [inline src/components/SocialMedias/index.js]: use map

File: src/components/Standings/index.js [hunk 10]
```diff
@@ -1,41 +1,44 @@
...
 
       <div>
-        <div className="Standings-Buttons">
+        <ButtonsContainer>
           <button
             style={{
-              backgroundColor: `${conference === "East" ? "black" : "white"}`,
-              color: `${conference === "East" ? "white" : "black"}`,
+              backgroundColor: `${conference === 'East' ? 'black' : 'white'}`,
+              color: `${conference === 'East' ? 'white' : 'black'}`
```
Comment (NareOP) [inline src/components/Standings/index.js]: remove inline styles for code readability, do it in styled-components with props

**Review by NareOP — CHANGES_REQUESTED** [review]
Good work, but be attentive on how you show the data, first of all look at the web page and separate similar elements in your mind, make components and their styling dynamic as much as possible.

**Pull request merged**